### PR TITLE
Get camera by cameraID

### PR DESCRIPTION
**Pull request by jaydenC88**

### CODE REVIEW

File: pythonAPIClient/client.py [hunk 1]
```diff
@@ -40,6 +40,21 @@ class Client(object):
 
     """
 
+    def _check_token(self, response, flag, url=base_URL, data=None, params=None):
```
Comment (mush-zhang) [inline pythonAPIClient/client.py]: to sync with master, remove the default value of url

**Review by mush-zhang — CHANGES_REQUESTED** [review]

**Comment by mush-zhang** [comment]
there is linting error that prevent it from passing circleci. also, could you merge master onto the branch? thank you!

File: pythonAPIClient/client.py (hunk 1 above)
Comment (mush-zhang) [reply pythonAPIClient/client.py]: merging master onto the branch and resolve confilcts would solve the problem

**Review by mush-zhang — APPROVED** [review]

File: pythonAPIClient/client.py [hunk 2]
```diff
@@ -308,7 +309,38 @@ def update_camera(self, camID, others):
...
+                raise AuthenticationError(response.json()['message'])
+            elif response.status_code == 404:
+                raise ResourceNotFoundError(response.json()['message'])
+            elif response.status_code == 403:
+                raise AuthorizationError(response.json()['message'])
+            elif response.status_code == 422:
+                raise FormatError(response.json()['message'])
+            else:
+                raise InternalError()
+        return response.json()
```
Comment (mush-zhang) [inline pythonAPIClient/client.py]: we should return a camera object instead

**Review by mush-zhang — CHANGES_REQUESTED** [review]

**Review by mush-zhang — APPROVED** [review]

**Review by Sripathm2 — APPROVED** [review]
grt job

**Review by Mrjimjr — APPROVED** [review]
Looks good. Your Test function names are descriptive. I don't think any docstrings are needed.

**Pull request merged**